spec/lib/configuration_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+module FloodRiskEngine
+  describe Configuration do
+    it "adds #config to the engine module" do
+      expect(FloodRiskEngine).to respond_to(:config)
+    end
+
+    it "has the correct default values" do
+      expect(FloodRiskEngine.config.test_me).to eq "test"
+    end
```

Comment:
Or change the test to so it test the expected response when you call a non-existent config item.

Actually, that's the better option.